index.html

```diff
@@ -11,19 +11,11 @@
       <div class="d-flex justify-center mt-5 w-100">
         <div class="w-100">
           <header class="my-4"></header>
-          <main class="mt-10 d-flex justify-center">
-            <div class="d-flex flex-col">
-              <div class="d-flex justify-center">
-                <img src="./src/images/subway_emoji.png" width="200" />
-              </div>
-              <p class="mt-0 text-center">
-                지하철 노선도 앱을 사용하기 위해서는 로그인이 필요합니다.
-              </p>
-            </div>
-          </main>
+          <main class="mt-10 d-flex justify-center"></main>
+          <div class="modal"></div>
         </div>
       </div>
     </div>
-    <script type="module" src="./src/js/index.js"></script>
+    <script src="./src/js/index.ts"></script>
```

Comment:
```suggestion
    <script src="./src/index.ts"></script>
```
js라고 하기엔 ts를 사용하고 있어서 폴더 명이 어색한 것 같아요! 아예 src에서 불러오거나 혹은 scripts 라는 폴더를 만들어서 관리해주면 어떨까요?

Reply:
아 그렇네요! 폴더명이라던지 구조에 대해서 신경을 많이 못썼던 것 같아요ㅜㅜ